mopidy_local_images/library.py

```diff
@@ -228,3 +219,6 @@ def _scan(self, uri):
         logger.debug('Scanning %s for images', uri)
         data = self.scanner.scan(uri)
         return data.get('tags', {})
+
+    def get_data_dir(self, config):
+        raise NotImplementedError  # overridden by Extension
```

Comment:
I don't think there's anything the migrate function needs from the Extension. Might as well just move it here.